js/components/graph/Sidebar.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import Focus from "./Focus";
+import PropTypes from "prop-types";
+
 
 export default class Sidebar extends React.Component {
   constructor(props) {
     super(props);
+    this.reset;
```

Comment:
This isn't needed because you're calling the reset method from the `Graph` component